contract/src/counter.compact

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pragma language_version 0.16;
+pragma language_version 0.17;
```

Comment:
I'd recommend using `>= 0.16 && <= 0.17` instead since there's no breaking changes between `0.16` and `0.17` and if someone hasn't updated their compactc this code should still compile.